translations/texts/objects/apex/apexbed/apexbed.object.json

```
@@ -68,7 +68,7 @@
     },
     "Texts": {
       "Eng": "Sssuch uncomfortable bedding. No match for Floran fursss.",
-      "Rus": ""
+      "Rus": "Такое неудобное постельное белье. Не подходит для мехов Флоран."
```

```suggestion
      "Rus": "Такое неудобное постельное белье. Ни чета флоранским мехам."
```
Флораны совсем не из меха сделаны, они ходячие растения. Но спят на мехах.